target/linux/ath79/image/common-compex.mk

```diff
@@ -0,0 +1,7 @@
+define Device/compex_wpj-16m
+# TODO wpj563 usb3
+  ATH_SOC := qca9531
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-ledtrig-usbport
+  MTDPARTS := spi0.0:192k(u-boot)ro,16128k(firmware),64k(art)ro
+  IMAGE_SIZE := 16128k
+endef
```

Comment:
I think the 3 remaining lines do not justify a common partition for these devices.
Just add them to the individual devices which you currently have in generic-compex.mk and probably move to generic.mk.
Just remove common-compex.mk then ...

Reply:
i need a few minutes to fix the commits